### ISSUE TITLE
I think the API is down

### ISSUE THREAD

**Issue by HarleyGolfGuy** [pro-react/kanban-app#27]
When running the kanban-app I get: 

OPTIONS http://kanbanapi.pro-react.com/cards 404 (Not Found)

And when I go to the same URL in a browser, the page says:

"There is nothing here yet.  Build something amazing"

**Comment by yGUvMu7uWPbUdJHk** [comment]
Yeah I found this too on Friday but my comment seems to have disappeared. Also I can't find the API server to use as an alternative (although I'd prefer to use the API).

**Comment by HarleyGolfGuy** [comment]
Ok it works now, I don't know for certain, but you may have to go to http://kanbanapi.pro-react.com first and accept the terms...maybe my authorization expired?  Anyhow, it's back up and working for me!

**Comment by lingkongc** [comment]
Where is the local API server? I can't find it.
